templates/500.html

```diff
@@ -1,22 +1,20 @@
 {% extends 'base.html' %}
 
 {% block content %}
-    <section class="error-server">
-            <div class="error-server__box-img">
-                <div class="container">                                   
-                    <h1 class="error-server__heading">Oops!<br> 500</h1>
-                    <h3 class="error-server__text">
-                        It looks like that there is an internal server error.
-                    </h3>
-                    <!-- Href Home Button -->
-                    <div> 
-                        <button class="button">
-                            <a href="index.html" aria-label="Back Home!">
-                                Back Home!
-                            </a>
-                        </button>
-                    </div>
-                </div>                
+    <section class="error-server">            
+        <div class="error__container error__box-img">                                   
+            <h1 class="error__heading">Oops!<br> 500</h1>
+            <h3 class="error__text">
+                It looks like that there is an internal server error.
+            </h3>
+            <!-- Href Home Button -->
+            <div> 
+                <button class="error__button error__button--server">
+                    <a href="index.html" aria-label="Back Home!">
```

Comment:
And one to please 😉 This href attribute value needs to be:
```href="{{url_for('index')}}"``` 

This is the only way a href will work with Flask 😁